### PR TITLE
json-unit 2.32.0

### PR DESCRIPTION
**Pull request by sullis**

### CODE REVIEW

**Comment by timtebeek** [comment]
Perhaps we can introduce a version property if these two dependencies should be bumped in sync; I'm seeing #1813 and #1814 separately; might be nice to fix that properly in this PR.

**Comment by timtebeek** [comment]
Might also be good to go for 2.32.0, rather than debugging 2.31.0

File: build.gradle [hunk 1]
```diff
@@ -30,6 +30,7 @@ project.ext {
     guava          : '31.0.1-jre',
     jackson        : '2.13.1',
     xmlUnit        : '2.9.0',
+    jsonUnit       : '2.31.0',
```
Comment (timtebeek) [inline build.gradle:33]:
```suggestion
    jsonUnit       : '2.32.0',
```

File: build.gradle [hunk 2]
```diff
@@ -59,7 +60,7 @@ dependencies {
     exclude group: 'junit', module: 'junit'
   }
   api "org.xmlunit:xmlunit-placeholders:$versions.xmlUnit"
-  api 'net.javacrumbs.json-unit:json-unit-core:2.30.0'
+  api 'net.javacrumbs.json-unit:json-unit-core:$versions.jsonUnit'
```
Comment (timtebeek) [inline build.gradle:63]:
```suggestion
  api "net.javacrumbs.json-unit:json-unit-core:$versions.jsonUnit"
```

File: build.gradle [hunk 3]
```diff
@@ -97,7 +98,7 @@ dependencies {
   testImplementation "org.hamcrest:hamcrest-library:2.2"
   testImplementation 'org.mockito:mockito-core:4.3.1'
   testImplementation 'org.mockito:mockito-junit-jupiter:4.3.1'
-  testImplementation 'net.javacrumbs.json-unit:json-unit:2.30.0'
+  testImplementation 'net.javacrumbs.json-unit:json-unit:$versions.jsonUnit'
```
Comment (timtebeek) [inline build.gradle:101]:
```suggestion
  testImplementation "net.javacrumbs.json-unit:json-unit:$versions.jsonUnit"
```

Comment (timtebeek) [reply build.gradle:101]: Interpolation requires double quotes, which is why you're getting build failures.

File: build.gradle (hunk 2 above)
Comment (timtebeek) [reply build.gradle:63]: You'll want to apply this one as well

Comment (sullis) [reply build.gradle]: Done.

**Comment by sullis** [comment]
Ready for review @timtebeek

**Review by timtebeek — APPROVED** [review]

**Comment by timtebeek** [comment]
@jamietanna are you able to see this merged?
Should close #1813 & #1814 as well

**Review by jamietanna — APPROVED** [review]

**Comment by jamietanna** [comment]
Thanks for this :raised_hands:

**Pull request merged**